ams/server/src/main/java/com/netease/arctic/server/optimizing/maintainer/IcebergTableMaintainer.java

```diff
@@ -207,7 +248,65 @@ private void expireSnapshots(long olderThan, Set<String> exclude) {
       parentDirectory.forEach(
           parent -> TableFileUtil.deleteEmptyDirectory(arcticFileIO(), parent, exclude));
     }
-    LOG.info("to delete {} files, success delete {} files", toDeleteFiles.get(), deleteFiles.get());
```

Comment:
Should we print statistical information about the execution results to make it easier to find this information through logs?

Reply:
Sure, I thought there will print duplicate logs if user enable logging event listener. Should we consider avoiding deplicate logs?

Reply:
Yes, we should try to avoid printing duplicate logs. Perhaps we should limit the log event listener to only print Iceberg report events for now? The expiring process should print its own logs instead of relying on the event listener, as they are optional. What do you think?

Reply:
Do you mean that each event listener can only handle its own event type? Then we can limit the logging event listener only print `ICEBERG_REPORT`? @zhoujinsong

Reply:
Yes, that is what I expect.

Reply:
Ok. We may add some event filter options for `LoggingEventListener`, but we can add it in another PR.